libs/librepcbproject/erc/ercmsglist.h

```diff
@@ -54,9 +54,12 @@ class ErcMsgList final : public QObject, public IF_XmlSerializableObject
     public:
 
         // Constructors / Destructor
+        ErcMsgList() = delete;
+        ErcMsgList(const ErcMsgList& other) = delete;
         explicit ErcMsgList(Project& project, bool restore, bool readOnly, bool create) throw (Exception);
         ~ErcMsgList() noexcept;
 
+
```

Comment:
It would be perfect if you moved this empty line from here to above the "signals:" statement :)